portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,13 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Kenneth Kenfack's Portfolio</h1>
+      <p>This is my portfolio. It's all about me a Cameroonian king. Growing up to be amazing. I'm currently a sophmore at umbc</p>
+      <p> I love going on adventures, trying new things, track and field, video games, basketball </p>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+      <img src = "/images/IMG_2768rot.jpg"/>
```

Comment:
The photo is not in this commit? Ideally, each commit (particularly when working with others) is a stand-alone known to work state of the codebase. (Or bisection will fail.)

Reply:
You probably forgot to `git add` the photograph? `git status` will tell you this.